### PR TITLE
feat(webrenderer): merge webrenderer repo to hippy repo

### PR DESCRIPTION
**Pull request by pba-cra**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2609) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2609) before we can accept your contribution.<br/>**11** out of **12** committers have signed the CLA.<br/><br/>:white_check_mark: lavnFan<br/>:white_check_mark: zoomchan-cxj<br/>:white_check_mark: medns<br/>:white_check_mark: wwwcg<br/>:white_check_mark: ozonelmy<br/>:white_check_mark: churchill-zhang<br/>:white_check_mark: shenchaoran<br/>:white_check_mark: siguangli<br/>:white_check_mark: crazyming9528<br/>:white_check_mark: iPel<br/>:white_check_mark: pba-cra<br/>:x: dependabot[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=2609) it.</sub>

**Review by zoomchan-cxj — COMMENTED** [review]
hippy-vue-next-style-parser 也加一下   
 "ae-wrong-input-file-type": {
        "logLevel": "none"
      }

File: packages/hippy-web-renderer/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@hippy/web-renderer",
+  "version": "0.0.4",
```
Comment (zoomchan-cxj) [inline packages/hippy-web-renderer/package.json:3]: 0.0.4 改成 unspecified

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**